neptune/new/exceptions.py

```diff
@@ -191,7 +238,26 @@ def __init__(
         )
 
 
-class ProjectNotFound(ExceptionWithProjectsWorkspacesListing):
+class ContainerUUIDNotFound(NeptuneException):
+    container_id: str
+    container_type: ContainerType
+
+    def __init__(self, container_id: str, container_type: ContainerType):
+        self.container_id = container_id
+        self.container_type = container_type
+        super().__init__(
+            "{} with ID {} not found. Could be deleted.".format(
+                container_type.value.capitalize(), container_id
+            )
+        )
+
+
+RunUUIDNotFound = ContainerUUIDNotFound
+
+
+class ProjectNotFoundWithSuggestions(
```

Comment:
is this the only class extending `ExceptionWithProjectsWorkspacesListing`? if yes, I'd prefer to put its logic here (if nothing else, so we don't have to think/worry about multiple inheritance...).

Reply:
ok, no discussion then